modules/canbus/conf/canbus_conf.pb.txt

```diff
@@ -1,5 +1,5 @@
 vehicle_parameter {
-  brand: LINCOLN_MKZ
+  brand: GE3
```

Comment:
Don't modify default configration file, please use another file for new vehicle.